main.py

```diff
@@ -1,13 +1,102 @@
 #!/usr/bin/env python
+import re
+from nltk.wsd import lesk
 
-# Your classes or methods go here:
 
+class JsonParsedWriter:
 
-class Tool:
-    def my_method(self):
-        print("my_method called!")
+    @staticmethod
+    def write(data, output_name, indent=2):
+        import json
```

Comment:
Avoid imports that are near to the place where they are used. Contrary to the local variables (that should be defined closer to the exact place where they are used), all imports should be at the top of your module.